app/js/components/JobRow.jsx

```diff
@@ -9,19 +10,23 @@ const JobRow = ({ data, onClick }) => (
             </a>
         </td>
         <td>
-            {data.started}
+            { moment.utc(data.started).format('M-D-Y hh:mm:ss') }
```

Comment:
Yeah it is off by 7 hours again. So for these we want just the normal constructor which considers the timezone. 
The date format should also be `YY-MM-DD hh:mm:ss` to match ISO 8601.